src/Kdyby/Translation/PrefixedTranslator.php

```diff
@@ -49,6 +50,14 @@ public function __construct($prefix, ITranslator $translator)
 
 	public function translate($message, $count = NULL, $parameters = array(), $domain = NULL, $locale = NULL)
 	{
+		if (Utils\Strings::startsWith($message, '../')) {
```

Comment:
I don't think I wanna support relative paths. I would be okey with supporting absolute paths. Something like.

```php
$prefixed->translate('//front.homepage.hello');
```

also, we don't need to check for `../` which are all single-byte characters with multibyte functions. plain old `stripos()` would be enough